.github/workflows/conformance-aks.yaml

```diff
@@ -143,21 +143,43 @@ jobs:
             az account show
 
       - name: Create AKS cluster
+        id: cluster-creation
         run: |
+          # Create group
           az group create \
             --name ${{ env.name }} \
             --location ${{ env.location }} \
             --tags usage=${{ github.repository_owner }}-${{ github.event.repository.name }} owner=${{ steps.vars.outputs.owner }}
+
+          # Create cluster with a 1 node-count (we will remove this node pool
+          #  afterwards)
```

Comment:
Nit: You have additional spaces between the comment marker (`#`) and the beginning of your text, here and at other locations in the YAML and the documentation files, but not everywhere. Is there a reason for that?

Reply:
I think I've added them to make it more readable. I'll revert it back